src/ida_lunde_naalsund_ex/ex_02/file_stats.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+
+__author__ = "Ida Lunde Naalsund"
+__email__ = "idna@nmbu.no"
+
+
+def char_counts(textfilename):
+    """Takes name of text file as input and
+    counts the occurrences of each character
+    in the text.
+
+    :param textfilename: Name of file which
+    characters we want to count
+    :return: List with character codes with their
+    corresponding number of occurrences.
+    """
+    text = open(textfilename).read()
```

Comment:
don't do this, either assign a variable to the file, then call `.read()` on that variable and subsequently close it by `.close()` when your done with it. Or the preffered way is using a `while` block.
The reason this is bad is that there is no handle to the file. Python will most probably close it in garbage collection, but it is bad practice in general.